tests/x11/reboot_gnome.pm

```diff
@@ -19,7 +19,7 @@ use utils;
 sub run {
     my ($self) = @_;
     # 'keepconsole => 1' is workaround for bsc#1044072
-    power_action('reboot', keepconsole => 1);
+    power_action('reboot', keepconsole => 1, s390_reboot => get_var('S390_ZKVM', 0));
```

Comment:
I would suggest something to explicitly emphasize that it's about any reboot except initial. E.g. reboot_installed_system_zkvm . Just s390 can be be misleading as includes zVM.